packages/anywidget/src/widget.js

```diff
@@ -98,6 +98,39 @@ async function load_esm(esm) {
 	return widget;
 }
 
+/**
+ * @param {import("@jupyter-widgets/base").DOMWidgetView} view
+ * @returns {import("@anywidget/types").RenderContext}
+ *
+ * Prunes the view down to the minimum context necessary for rendering.
+ * Calls to `model.get` and `model.set` automatically add the view as
+ * context to the model, so we can gracefully unsubscribe from events
+ * added by the user-defined render.
+ */
+function extract_context(view) {
+	let model = {
```

Comment:
lgtm

I suppose you could "leak" the `AnyView` instance as a private `_view` member of the model proxy in case there is exceptional use case for it, but it might make more sense to wait for someone to ask for functionality the model proxy can't provide.

Reply:
I like the idea of waiting until/if there is a use case to expose the view in the render context. With this PR in place, very few jupyter-widget/backbone things are exposed to end users. It's a much narrower API that we need to support, which also allows us greater flexibility to change internals. 

For example, I have an idea for how to support custom serializers with this API.

Reply:
This removes model.send which I need. Can you bring that back?

Reply:
Oh no, sorry about that! Fixed in #146 and released in v0.5.2.